Manager/src/test/java/io/raspberrywallet/manager/ConfigurationTest.java

```diff
@@ -0,0 +1,82 @@
+package io.raspberrywallet.manager;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+class ConfigurationTest {
+
+    @Test
+    void fromYamlFile() throws IOException {
+        String version = "0.5.0";
+        long sessionLength = 3600000;
+        String basePrefixDir = "/opt/wallet";
+        String configYamlContent = "" +
```

Comment:
we can store reources for tests in test resources, analogically as src resources, this huge string is kinda.. big and chaotic :D it would be better to read it from file

Reply:
I decided to make it as is, for 2 reasons. 
First is that I can then easly comare the values like 
`assertEquals(configuration.getVersion(), version);`
How can I do that with `config.yaml` stored in `/res` directory ? It would be a totally mess.

Second is that my approach is independent, so everything that can change is in one  self contained function.

This is the only one place where this code/config is used so there is no need to encapsulate it.

Reply:
In your approach you need to make a change in two places which is always bad idea.

Reply:
Okay, I understand, then just remove my changes